data/schemas/keyboard.jsonschema

```diff
@@ -439,6 +440,14 @@
                         "timeout": {"$ref": "qmk.definitions.v1#/unsigned_int"}
                     }
                 }
+                "hand": {
+                    "type": "object",
+                    "additionalProperties": false,
+                    "properties": {
+                        "pin": {"$ref": "qmk.definitions.v1#/mcu_pin"},
```

Comment:
this option (SPLIT_HAND_PIN) also appears to be covered by info.py, as `split.main`
but that doesn't seem to work for me, the schema definition does have a split.main,
which has a `pin` option, but no way to set which pin to use

the associated low_is_left option seems to be missing from any merged implementations

Reply:
Looks like [`_extract_split_main` in `lib/python/qmk/info.py`](https://github.com/qmk/qmk_firmware/blob/6e1c2b20e55517cc70a0bfb37523ae7dbe26a7d8/lib/python/qmk/info.py#L296-L344) must be fixed then (it extracts the pin info from `SPLIT_HAND_MATRIX_GRID`, but not from `SPLIT_HAND_PIN`). The problem is that if we take the existing `SPLIT_HAND_MATRIX_GRID` → `split.matrix_grid` conversion as an example, the new property would be named `split.pin`, and I'm not sure that would be understandable.

Reply:
haven't looked further at this, but it seems like my approach is the clearer one, mostly by merit of existing...

would this PR be merged if i pull out the SPLIT_HAND_PIN part?

Reply:
for split handedness, there is a larger PR in the works: https://github.com/qmk/qmk_firmware/pull/18254